### DISCUSSION TITLE
Github

### DISCUSSION THREAD

**Discussion by Darry96** [Launch-X-Latam/MisionFrontEnd#281] [Q&A]
Hola tengo una duda saben como puedo agrupar carpetas que ya tengo en mi repositorio de github a una nueva para tener mejor orden?

**Comment by doguedogue** [comment]
Es como el explorador de windows, puedes basarte en como está estructurado el repositorio de la misión de front end:
https://github.com/Launch-X-Latam/MisionFrontEnd

![image](https://user-images.githubusercontent.com/23409026/198899571-69ea8149-3161-41a4-9b99-436d77988756.png)

Cuando le das añadir archivo, puedes añadir carpetas escribiendo el nombre del folder/carpeta y luego la diagonal "/ ", esto crea la carpeta y luego agregas el archivo (si no agregas un archivo al final, no se agregan los folders vacios) de esa forma puedes añadirlos y darle commit (luego si quieres puedes borrar el archivo).

![image](https://user-images.githubusercontent.com/23409026/198899633-74f857cb-e9ad-4e05-9c71-40d9188902af.png)

![image](https://user-images.githubusercontent.com/23409026/198899728-568414d9-2d14-4cff-8b9a-27e0995982e7.png)

El archivo que se muestra en una carpeta por default es el README.md, es como el index.html de las páginas web. normalmente se agrega uno por carpeta para que describas que hay ahi.

En las carpetas puedes pegar imagenes que tengas en el portapapeles ( como una impresión de pantalla) o bien la puedes referenciar con notación markdown (archivos md) para agregarlos como los links (anchor), asi:

```![Nombre de imagen](images/imagen.jpg)```
